tensorflow/python/keras/optimizer_v2/learning_rate_schedule.py

```diff
@@ -599,9 +600,10 @@ def __call__(self, step):
       completed_fraction = global_step_recomp / decay_steps
       cosine_decayed = 0.5 * (1.0 + math_ops.cos(
           constant_op.constant(math.pi) * completed_fraction))
+      thetha = initial_learning_rate - self.alpha
+      decayed = math_ops.multiply(thetha, cosine_decayed)
+      return math_ops.add(decayed, self.alpha)
```

Comment:
This seems to change the value that gets computed.

Reply:
It might slighlt different. The point of my commit is to implement a method according to its [original paper](https://paperswithcode.com/method/cosine-annealing) since the last method had slightly different results. Also, i wonder whats the differences between `convert_to_tensor_v2_with_dispatch` and  `convert_to_tensor_v2`.